lib/Bio/EnsEMBL/Mongoose/Serializer/RDFCoordinateOverlap.pm

```diff
@@ -0,0 +1,101 @@
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+=cut
+
+package Bio::EnsEMBL::Mongoose::Serializer::RDFCoordinateOverlap;
+
+use Moose;
+use Bio::EnsEMBL::Mongoose::IOException;
+use URI::Escape;
+extends 'Bio::EnsEMBL::Mongoose::Serializer::RDFLib';
+
+has handle => ('is' => 'ro', required => 1, isa => 'Ref');
+
+
+sub print_coordinate_overlap_xrefs {
+  my $self = shift;
+  my $ens_id = shift;
+  my $record = shift;
+  my $source = shift;
+  my $score = shift;
+
+  my $fh = $self->handle;
+  my $id = $record->id;
+  unless ($id) {$id = $record->primary_accession}
+  my $clean_id = uri_escape($id);
+  my $namespace = $self->identifier($source);
+
+  my $xref_source = $self->prefix('ensembl').$ens_id;
+  my $xref_link = $self->new_xref('ensembl',$source);
+
+  $namespace = $self->identifier($source);
+  $namespace = $self->prefix('ensembl').$source.'/' unless $namespace;
+  my $xref_target = $namespace.$clean_id;
+  # Meta about Ensembl ID
+  print $fh $self->triple($self->u($xref_source), $self->u($self->prefix('dcterms').'source'), $self->u($self->prefix('ensembl')));
+  print $fh $self->triple($self->u($xref_source),$self->u($self->prefix('dc').'identifier'), qq/"$ens_id"/);
+  print $fh $self->triple($self->u($xref_source), $self->u($self->prefix('rdfs').'label'), '"'.$ens_id.'"' );
+  # Create link
+  print $fh $self->triple($self->u($xref_source), $self->u($self->prefix('term').'refers-to'), $self->u($xref_link));
+  print $fh $self->triple($self->u($xref_link), $self->u($self->prefix('term').'refers-to'), $self->u($xref_target));
+  # Annotate link
+  print $fh $self->triple($self->u($xref_link),$self->u($self->prefix('rdf').'type'),$self->u($self->prefix('term').'Coordinate_overlap'));
+  print $fh $self->triple($self->u($xref_target),$self->u($self->prefix('dc').'identifier'), qq/"$id"/);
+  print $fh $self->triple($self->u($xref_link),$self->u($self->prefix('term').'score'),qq/"$score"/);
+}
+
+sub print_slimline_coordinate_overlap_xrefs {
+  my $self = shift;
+  my $ens_id = shift;
+  my $record = shift;
+  my $source = shift;
+
+  my $fh = $self->handle;
+  my $id = $record->id;
+  unless ($id) {$id = $record->primary_accession}
+  my $clean_id = uri_escape($id);
+  my $namespace = $self->identifier($source);
+
+  my $xref_source = $self->prefix('ensembl').$ens_id;
+  my $xref_link;
+
+  $namespace = $self->identifier($source);
+  $namespace = $self->prefix('ensembl').$source.'/' unless $namespace;
+  my $xref_target = $namespace.$clean_id;
+
+  print $fh $self->triple($self->u($xref_source), $self->u($self->prefix('term').'refers-to'), $self->u($xref_target) );
+}
+
+
+# Write triples which attach source labels to sources, handy for pretty printing or maybe text search
```

Comment:
Blatant copy-paste is blatant. Since you've built a sister-serialiser, print_source_meta shouldn't be here. It's only needed once in any case. I'm looking for better ways to abstract some of the print calls so we can refactor this package away